packages/@okta/i18n/src/properties/login_nl_NL.properties

```diff
@@ -730,16 +741,21 @@ oie.authenticator.verify.error.fail = Kan authenticator niet verifi\u00EBren. Pr
 
 ## Password
 oie.password.label = Wachtwoord
+oie.password.newPasswordLabel = Nieuw wachtwoord
 oie.password.authenticator.description = Kies een wachtwoord voor uw account
 oie.password.challenge.title = Verifi\u00EBren met uw wachtwoord
 oie.password.enroll.title = Wachtwoord instellen
 oie.password.passwordLabel = Wachtwoord invoeren
 oie.password.confirmPasswordLabel = Wachtwoord opnieuw invoeren
 oie.password.reset.verification = Verifieer met een van de volgende authenticators om uw wachtwoord opnieuw in te stellen.
+idx.password.expiring.message = Wanneer je wachtwoord verloopt, wordt je Okta-account vergrendeld.
+oie.password.incorrect.message = Wachtwoord is onjuist
 
 ## Phone
 oie.phone.label = Telefoon
 oie.phone.authenticator.description = Verifieer met een code die naar uw telefoon wordt gestuurd
+oie.phone.enroll.sms.label = Sms
```

Comment:
Just checking if this the right translation for this label ? I see it as `SMS` in some other languages

Reply:
@nikhilvenkatraman-okta This is correct. Dutch capitalization convention doesn't follow English capitalization convention. In Dutch it's common to see "sms" (all lowercase) or "Sms" (initial uppercase 's'). Example: `https://support.google.com/messages/answer/6080324?hl=nl`